docs/migrating/backup.md

```diff
@@ -0,0 +1,59 @@
+# Backup 
+
+You should always make sure that your solution is properly backed up. The following example shows you how to do this on a Linux/UNIX based system where eZ Platform is using a MySQL database called "example". You should close the system before making a backup or an upgrade.
```

Comment:
`Linux-UNIX-based`
Not sure about `close the system`, sounds like closing the OS, is that what is meant? Or shut down Platform if it's running?